spec/jasmine-helper.js

```diff
@@ -0,0 +1,222 @@
+'use babel'
+/* @flow */
```

Comment:
@maxbrunsfeld  I wouldn't get too attached to `'use babel'`. I talked to @sebmck, and it seems that once Babel 6.x is out, we should be able to support `.babelrc` files in a sane way in Atom such that we don't need this pragma anymore.

As for `/* @flow */`, yes, apparently it has to be inside one of those right now: http://flowtype.org/docs/five-simple-examples.html. I'm following up to see if we can make this an option in `.flowconfig`.